validation-test/compiler_crashers_fixed/28855-swift-genericsignaturebuilder-addrequirement-swift-requirementrepr-const-swift-g.swift

```diff
@@ -5,5 +5,5 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not --crash %target-swift-frontend %s -emit-ir
+// RUN: not %target-swift-frontend %s -emit-ir
```

Comment:
Well, that's unexpected.

Reply:
Unexpected like huh interesting, or unexpected like you'd prefer I detour into figuring out why it was crashing in the first place?